src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/RuntimeController.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Helpers;
+using Altinn.Platform.Storage.Models;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Serilog;
+using Serilog.Core;
+
+namespace Altinn.Platform.Storage.Controllers
+{
+    [Route("/runtime/api/instances/{instanceId}/data")]
```

Comment:
Should this path follow the API route naming convention? 
/api/{component}/v1/{resource}

Reply:
Have moved RuntimeController to test